way_to_home/place/models.py

```diff
@@ -1 +1,68 @@
 """This module implements class that represents the place entity."""
+
+from user.models import User
+from django.db import models, IntegrityError
+from utils.abstract_models import AbstractModel
+
+
+class Place(AbstractModel):
+    """Model for place entity."""
+
+    longitude = models.DecimalField()
+    latitude = models.DecimalField()
+    name = models.CharField(max_length=64)
+    address = models.CharField(max_length=255)
+    stop_id = models.PositiveSmallIntegerField()
+    user = models.ForeignKey(User, on_delete=models.CASCADE, required=True)
+
+    def __str__(self):
+        """Method that returns route instance as string."""
+
+        return f'{self.name}'
```

Comment:
```suggestion
        return f'{self.name}'
```
it is not necessary to use f-strings in this case

Reply:
Done